### PR TITLE
Adding blog post relating to contribution sprint

### PR DESCRIPTION
**Pull request by willykaram**
Created a blog post and added some images relating to today's contribution sprint

### CODE REVIEW

File: posts/blog--contribution-sprint.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Contribution Sprint
+author: Chris McCord
```
Comment (jeregrine) [inline posts/blog--contribution-sprint.markdown]: Author is you Willy :-)

**Review by jeregrine — CHANGES_REQUESTED** [review]
One small change.

**Comment by willykaram** [comment]
I updated the author, but I think we should list @chrismccord or a Phoenix team member like you/@jeregrine or @Gazler  as an author -- please feel free to revise however you see fit.

**Comment by josevalim** [comment]
@willykaram you led the effort, you should definitely be the author. :)

**Comment by Gazler** [comment]
Thanks <3 <3 <3

**Pull request merged**